CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
       - FIXED #4781: Fixed overflow exceptions in percent-encoding parsing
     - Guidance:
       - CHANGED #4706: Guidance refactoring step to decouple intersection connectivity analysis and turn instructions generation [#4706](https://github.com/Project-OSRM/osrm-backend/pull/4706)
+      - CHANGED #4751: Guidance refactor to pull `isThroughStreet` check into one unified function [#4751](https://github.com/Project-OSRM/osrm-backend/issues/3491)
```

Comment:
Not sure how we currently create the changelog.
This changeset should be transparent to users.
Not sure if we want to announce it.